doc/data-mining-library/source/tutorial/data.rst

```diff
@@ -39,13 +39,13 @@ The rest of the table gives the data. Note that there are 5 instances in our tab
 
 Note that for the file name no suffix is needed, as Orange checks if any files in the current directory are of a readable type. The call to ``Orange.data.Table`` creates an object called ``data`` that holds your dataset and information about the lenses domain:
 
-    >>> data.domain.attributes
-    (DiscreteVariable('age', values=['pre-presbyopic', 'presbyopic', 'young']),
-     DiscreteVariable('prescription', values=['hypermetrope', 'myope']),
-     DiscreteVariable('astigmatic', values=['no', 'yes']),
-     DiscreteVariable('tear_rate', values=['normal', 'reduced']))
+    >>> data.domain.attribute()
```

Comment:
??

Reply:
PyCharm froze while I was typing, and I couldn't find what changed, so I assumed I didn't change anything.